Unified/checkor.py

```diff
@@ -1012,10 +1013,43 @@ def upward( ns ):
 
         time_point("checked custodiality", sub_lap=True)
 
-        ## presence in phedex
+        ## presence in phedex and/or rucio
         phedex_presence ={}
+        rucioClient = RucioClient()
```

Comment:
We may  put the `account=CMSUnified` either in the RucioConig, or in the code while calling the constructor. The later I didn't want to do, because you know  we are trying to separate the Prod and Testbed configurations, and I am trying not to put any more obstacles on the road.

Reply:
@sharad1126 BTW the needed variables to be exported may safely be put in the wrapper .sh script calling the checkor.py module

Reply:
@todor-ivanov I tried and it finally worked with `account='unified'`. So can you please make the changes and either change default config or make the call in the object creation with the parameters as `account='unified'`
Looks like we won't need the cfg file at all.

Reply:
```
[cmsunified@vocms0277 WmAgentScripts]$ python
Python 2.7.13+ (default, Dec 18 2018, 10:09:09) 
[GCC 6.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from RucioClient import RucioClient
/data/srv/wmagent/v1.3.0/sw/slc7_amd64_gcc630/external/py2-cryptography/2.4.2-comp2/lib/python2.7/site-packages/cryptography/hazmat/bindings/openssl/binding.py:163: CryptographyDeprecationWarning: OpenSSL version 1.0.1 is no longer supported by the OpenSSL project, please upgrade. A future version of cryptography will drop support for it.
  utils.CryptographyDeprecationWarning
>>> rucioClient = RucioClient(account='unified')
>>> 
>>> 
```

Reply:
AFAIK, the account has to be explicitly provided. So adding it to the config file won't help.

Reply:
@amaltaro we are actually not using the config file separately at all. We have the default config file in rucioclient.py where currently account is `None`. So it can either be changed to `unified` or can just make the object creation with 
` rucioClient = RucioClient(account='unified')`